IrinaLoginovaHomeWork_2_2.cpp

```diff
@@ -0,0 +1,38 @@
+#include <cmath>
+#include <cstdint>
+#include <iostream>
+#include <stdexcept>
+
+int main() {
+    int array_NubmerTicket[6];
+    constexpr int MinRange = 1, MaxRange = 9;
+    std::cout << "Please enter 6 digits of your ticket. Numbers ranging from "
+        << MinRange << " to " << MaxRange << std::endl;
+    for (size_t i = 0; i <= 5; ++i) {
+        std::cout << "Number is --> " << std::endl;
+        std::cin >> array_NubmerTicket[i];
+        while (array_NubmerTicket[i] == 0 || array_NubmerTicket[i] < MinRange ||
+            array_NubmerTicket[i] > MaxRange) {
+            std::cout << "You entered an invalid number. Repeat a number in the "
+                "correct range"
+                << std::endl;
+            std::cout << "Number is --> " << std::endl;
+            std::cin >> array_NubmerTicket[i];
+        }
+    }
+    int sum_1 = {};
+    int sum_2 = {};
```

Comment:
```suggestion
    int sum_1 {};
    int sum_2 {};
```
когда то лишний оператор присвоение может с вами сыграть злую шутку)

Reply:
Записала. Надо себя приучить. :-)